cmd/fpga_admissionwebhook/fpga_admissionwebhook_test.go

```diff
@@ -30,6 +31,115 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+func TestParseResourceName(t *testing.T) {
+	tcases := []struct {
+		input       string
+		interfaceID string
+		afuID       string
+		expectedErr bool
+	}{
+		{
+			input:       "intel.com/fpga-arria10",
+			expectedErr: true,
+		},
+		{
+			input:       "intel.com/fpga-unknown",
+			expectedErr: true,
+		},
+		{
+			input:       "example.com/fpga-something",
+			expectedErr: true,
+		},
+		{
+			input:       "intel.com/fpga-arria10-nlb0",
+			interfaceID: "ce48969398f05f33946d560708be108a",
```

Comment:
it's hard to understand which part is device and which is af. Can we have it like this "intel.com/fpga-arria10/nlb0" or similar?

Reply:
arria10 would be also nice to have capitalized. intel.com/fpga/Arria10/nlb0 ?

Reply:
we can do case insensitive matching.